blazor/rich-text-editor/how-to/update-value.md

```diff
@@ -14,40 +14,78 @@ To achieve this, the `keydown` event has to be bound to the Rich Text Editor con
 ```cshtml
 
 @using Syncfusion.Blazor.RichTextEditor
+@inject IJSRuntime JSRuntime
 
-<div class="control-section">
-    <SfRichTextEditor ID="defalt_RTE" ref="rteObj" onkeydown="@onKeydown" >
-        <RichTextEditorToolbarSettings Items="@tools"></RichTextEditorToolbarSettings>
-        <p>The Rich Text Editor component is WYSIWYG ('what you see is what you get') editor that provides the best user experience to create and update the content. Users can format their content using standard toolbar commands.</p>
-        <p><b> Key features:</b></p>
-        <ul>
-            <li><p> Provides <b>IFRAME</b> and <b>DIV</b> modes </p></li>
-            <li><p> Capable of handling markdown editing.</p></li>
-            <li><p> Contains a modular library to load the necessary functionality on demand.</p></li>
-            <li><p> Provides a fully customizable toolbar.</p></li>
-            <li><p> Provides HTML view to edit the source directly for developers.</p></li>
-            <li><p> Supports third - party library integration.</p></li>
-            <li><p> Allows preview of modified content before saving it.</p></li>
-        </ul>
+
+<div class="control-section"> 
+    <SfRichTextEditor ID="defalt_RTE" ref="editor" onkeydown="@onKeydown" @bind-Value="@RteValue">
+        <RichTextEditorToolbarSettings Items="@Tools"></RichTextEditorToolbarSettings>
     </SfRichTextEditor>
 </div>
+<script>
+    window.preventSaveAction = function () {
+        document.addEventListener("keydown", function (e) {
```

Comment:
Dont bind the keydown event to the document. Can you bind it to contenteditable div element? pass the element id from server to client and then bind the keydown event.

Reply:
addressed the review comments